jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/actions/DeleteResourceAction.kt

```diff
@@ -34,7 +34,7 @@ abstract class DeleteResourceAction<in T : AwsExplorerResourceNode<*>>(text: Str
                 Messages.getWarningIcon(),
                 null,
                 object : InputValidator {
-                    override fun checkInput(inputString: String?): Boolean = inputString == resourceName
+                    override fun checkInput(inputString: String?): Boolean = inputString == message("delete_resource.confirmation_text")
```

Comment:
Can you show in greyed out text "delete me"?

Reply:
Done!

Reply:
it doesnt look like we have access to the textbox (and also its a `JTextComponent` instead of a `JBTextField` so we can't set the empty text anyways), so I think we'd need to move away from `showInputDialog` to do that. we might have an example of this somewhere else though

Reply:
Check out `SchemaSelectionPanel` for how to build a UI with the UI dsl, then we can use a `DialogWrapper`